src/services/tx/tx-sender/create.ts

```diff
@@ -79,7 +79,13 @@ export const createExistingTx = async (
   // Create a tx and add pre-approved signatures
   const safeTx = await createTx(txParams, txParams.nonce)
   Object.entries(signatures).forEach(([signer, data]) => {
-    safeTx.addSignature({ signer, data, staticPart: () => data, dynamicPart: () => '' })
+    safeTx.addSignature({
+      signer,
+      data,
+      staticPart: () => data,
+      dynamicPart: () => '',
+      isContractSignature: false,
```

Comment:
What does this do? I can't find a reference in the protocol-kit repo

Reply:
The new alpha version supports EIP-1271. So a signature can be a contract signature (EIP-1271 Signature).
In that case the dynamic part would be filled etc.

I did not add 1271 support yet in this PR as this focused on Base Sepolia. That's why it is always false and we still sign on-chain with SmartContractWallets.